src/config/configmanager.cpp

```diff
@@ -383,11 +383,17 @@ bool ConfigManager::reload() {
 	return result;
 }
 
+void ConfigManager::missingConfigWarning(const char* identifier) {
+	g_logger().warn(fmt::format("{}: Missing configuration for identifier: {}", __FUNCTION__, std::string(identifier)));
+}
```

Comment:
I think, creating a function it's not necessary. You can call the logger directly.

Reply:
I did it this way to avoid repeating the code

Reply:
Your idea of avoiding code repetition seems valid, and if you want to keep it, I see no issue. However, the use of fmt format within the g_logger is not necessary, as spdlog already uses fmt internally by default.